### PR TITLE
cyberjag | added the quality parameter for jpg

### PR DESCRIPTION
**Pull request by cyberjag**
Added a quality parameter in the option for jpg images.

### CODE REVIEW

**Comment by fiantinangon** [comment]
Ah i love this PR, we need the quality option to increase performance when processing large image 👌

File: src/index.js [hunk 1]
```diff
@@ -25,7 +26,7 @@ module.exports = async function({
   }
   await page.setContent(html)
   const element = await page.$('body')
-  const buffer = await element.screenshot({ path: output, type, omitBackground: transparent, encoding })
+  const buffer = await element.screenshot({ path: output, type, quality: quality, omitBackground: transparent, encoding })
```
Comment (frinyvonnick) [inline src/index.js:29]: if the variable name is the same you don't need to repeat it 👍 

```suggestion
  const buffer = await element.screenshot({ path: output, type, quality, omitBackground: transparent, encoding })
```

Comment (cyberjag) [reply src/index.js]: @frinyvonnick thanks for the feedback. Fixed the repetition.

**Comment by frinyvonnick** [comment]
Thank you for your work 🙏 I'll publish it in a further version.

**Comment by frinyvonnick** [comment]
This has been merged in [2.1.0](https://github.com/frinyvonnick/node-html-to-image/releases/tag/v2.1.0) 🚀

**Pull request merged**